config/datatypes_conf.xml.sample

```diff
@@ -187,6 +187,7 @@
     <datatype extension="searchgui_archive" type="galaxy.datatypes.binary:SearchGuiArchive" display_in_upload="True"/>
     <datatype extension="peptideshaker_archive" type="galaxy.datatypes.binary:CompressedArchive" subclass="True" display_in_upload="True"/>
     <!-- End Proteomics Datatypes -->
+    <datatype extension="netcdf" type="galaxy.datatypes.binary:NetCDF" mimetype="application/octet-stream" display_in_upload="true" description="Format used by netCDF software library for writing and reading chromatography-MS data files." />
```

Comment:
I knew this feature, but I didn't want to interfere with existing datatypes or to consume machine time by "parsing" all binaries.
But ok, let do that!